lib/order.rb

```diff
@@ -0,0 +1,65 @@
+class Order
+
+  attr_reader :current_order, :bill, :string_summary
+
+  def initialize
+    @menu = Menu.new
+    @current_order = []
+    @bill = 0
+    @string_summary = ''
+  end
+
+  def order_received(food_order, quantity = 1 )
+    raise "There is no #{food_order} available on the menu" unless on_menu?(food_order)
+    save_order(food_order, quantity)
+    "Order received for #{quantity} #{food_order}."
+  end
+
+  def check_total
+    raise "There is an error - your bill is not the same as the item cost." unless bill_correct?
+    to_two_decimals = sprintf('%.2f', @bill)
```

Comment:
Is it possible to extract the code that converts a number to 2 decimal places to a separate method for clarity?